clients/admin-ui/src/features/config-wizard/SuccessPage.tsx

```diff
@@ -49,7 +50,21 @@ const SuccessPage = ({
 
   const onFinish = () => {
     dispatch(setActiveSystem(undefined));
-    return features.plus ? router.push("/datamap") : router.push("/system");
+    const basePath = router.basePath || "/";
+
+    const datamapRoute = resolveLink({
+      href: "/datamap",
+      basePath,
+    });
+
+    const systemRoute = resolveLink({
+      href: "/system",
+      basePath,
+    });
```

Comment:
I don't think `systemRoute` needs to be resolved since it'll be within the same zone as the config wizard

Reply:
Ah, right right